core/src/components/textarea/textarea.scss

```diff
@@ -136,6 +157,12 @@
   pointer-events: none;
 }
 
+:host([auto-grow]) .cloned-input {
+  // Remove when deprecating iOS 13 support
+  // Workaround for iOS 13 to avoid rendering a white box
+  // when scroll assist is enabled and a user taps on a textarea with auto-grow.
```

Comment:
I was also able to reproduce this issue on iOS 15, so it may not be iOS 15-specific.